react/features/speaker-stats/components/SpeakerStatsItem.js

```diff
@@ -0,0 +1,79 @@
+import leftPad from 'left-pad';
+import React, { Component } from 'react';
+
+import {
+    getHoursCount,
+    getMinutesCount,
+    getSecondsCount
+} from '../../base/util/timeUtils';
+
+/**
+ * React component for speaker stats item.
+ *
+ * @extends Component
+ */
+class SpeakerStatsItem extends Component {
+    /**
+     * SpeakerStatsItem component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        displayName: React.PropTypes.string,
+        hasLeft: React.PropTypes.boolean,
+        isDominantSpeaker: React.PropTypes.boolean,
+        speakerTime: React.PropTypes.number,
+        stats: React.PropTypes.object
+    }
+
+    /**
+     * Returns a string counting the total amount hours, minutes,
+     * and seconds passed.
+     *
+     * @param {int} milliseconds - Time total in milliseconds to convert.
+     * @returns {string} HH:MM:SS formatted.
+     */
+    formatDate(milliseconds) {
```

Comment:
I'd opt for a more human-friendly representation such as what momentjs has to offer: http://momentjs.com/docs/#/durations/